packages/fast-components-react-msft/app/app.tsx

```diff
@@ -36,14 +39,21 @@ const hypertextStyles: ComponentStyles<IHypertextClassNameContract, undefined> =
     }
 };
 
-export interface IAppState {
+enum Themes {
```

Comment:
Nit:  `Themes` is plural, yet, `Direction` is singular.  Shouldn't `Themes` be `Theme` for consistency?  The usage would then read `Theme.light` not `Themes.light` unless there will be multiple light themes?

Reply:
Changed to `Theme` to align